README.md

```diff
@@ -2,7 +2,7 @@
 
 # docker
 
-Avalon on [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/), ideal for beginners and studios alike.
```

Comment:
We do need to be explicit about Toolbox, as opposed to "For Windows", as Toolbox is more hidden, not as obvious to install. And this only works with Toolbox.

Reply:
Yeah, but we are linking to the Windows installation. Replaced that with the Windows and Mac installation instructions.